common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -2331,6 +2331,16 @@
   "function.wynntils.timeSinceLastKill.argument.includeShared": "Include Shared kills",
   "function.wynntils.timeSinceLastKill.description": "The time, in milliseconds, since you last killed a mob",
   "function.wynntils.timeSinceLastKill.name": "Time Since Last Kill",
+  "function.wynntils.toBackgroundText.argument.backgroundColor": "Color of the background",
+  "function.wynntils.toBackgroundText.argument.leftEdge": "Style of the left edge. NONE or PILL",
+  "function.wynntils.toBackgroundText.argument.rightEdge": "Style of the right edge. NONE or PILL",
+  "function.wynntils.toBackgroundText.argument.text": "Text to convert. Incorrect characters will be omtited",
+  "function.wynntils.toBackgroundText.argument.textColor": "Color of the text",
+  "function.wynntils.toBackgroundText.description": "Converts the text to use Wynncraft background style.",
+  "function.wynntils.toBackgroundText.name": "To Background Text",
+  "function.wynntils.toFancyText.argument.text": "Text to convert. Incorrect characters will be omtited",
```

Comment:
Typo. Also, I don't think the word "incorrect" is proper here. Better perhaps with "unsupported". (Also on line 2337).